js/src/util.js

```diff
@@ -64,7 +77,11 @@ const Util = (() => {
     },
 
     supportsTransitionEnd() {
-      return Boolean(transition)
+      if (transition === null) {
```

Comment:
Here you add a check if transition is set or not, you should revert this change it was better before that, because they aren't any check done each time we call this method

Reply:
Cyclic import (util -> eventHandler -> util) was causing me troubles yesterday, but now seems working correctly. Probably it was another error.